modules/cli/src/main/scala/scala/cli/commands/Run.scala

```diff
@@ -73,10 +78,16 @@ object Run extends ScalaCommand[RunOptions] {
         partial = None,
         postAction = () => WatchUtil.printWatchMessage()
       ) { res =>
+        for (process <- processOpt)
+          ProcUtil.interruptProcess(process)
```

Comment:
Maybe we should check if the process if running right before calling that.

When verbosity is increased, we should also log that we are sending a SIGINT to the process (we could even print the command we run in debug mode, just in case).

If the command to kill the process fails, we should maybe ignore the failure if the process isn't running anymore (might mean it exited between the first check and the call of the command to kill it).